### PR TITLE
Feature/to throw or not to throw

### PR DESCRIPTION
**Pull request by doozMen**

### CODE REVIEW

File: iOS/Swift/README.md [hunk 1]
```diff
@@ -593,6 +594,9 @@ extension Object { // ! no space between Mark and extension!
 }
 
 ````
+
+## To Throw or Not
+
```
Comment (fousa) [inline iOS/Swift/README.md:639]: Le quoi?

Comment (doozMen) [reply iOS/Swift/README.md:639]: Ik heb dit in een volgende push aangepast naar "`Throw` versus `return nil`"

File: iOS/Swift/README.md [hunk 2]
```diff
@@ -596,6 +597,46 @@ extension Object { // ! no space between Mark and extension!
...
+As en example look at looking up a model for a given `indexPath`.
+
+```swift
+enum ModelError: Error {
+	case notFound
+}
+class ViewModel {
+	func model(for indexPath: IndexPath) throws -> Country {
+				 guard indexPath.row > 0 else {
```
Comment (fousa) [inline iOS/Swift/README.md:615]: There is something wrong with your tab setting...

File: iOS/Swift/README.md [hunk 3]
```diff
@@ -596,6 +597,46 @@ extension Object { // ! no space between Mark and extension!
...
+
+```swift
+func cellForRowAtIndexPath... {
+	do {
+		let cell = list.dequeu("modelCell", indexPath)
+		let model = try viewModel.model(for: indexPath)
+		cell.titleLabel.text= model.title
+		return cell
+	} catch {
+		return list.dequeu("emptyCell", indexPath)
```
Comment (fousa) [inline iOS/Swift/README.md]: I think `dequeu` is written as `dequeue`.

File: iOS/Swift/README.md [hunk 4]
```diff
@@ -596,6 +597,46 @@ extension Object { // ! no space between Mark and extension!
...
+}
+
+```
+
+Use in datasourde delegate, can be used for `UICollectionView & UITableView`, method name below is abriviated.
+
+```swift
+func cellForRowAtIndexPath... {
+	do {
+		let cell = list.dequeu("modelCell", indexPath)
```
Comment (fousa) [inline iOS/Swift/README.md]: I think dequeu is written as dequeue.

**Review by fousa — COMMENTED** [review]
Nice addition! 🤡

**Pull request merged**